aviasales/src/app/components/card/card.component.html

```diff
@@ -0,0 +1,60 @@
+<div class="card">
+  <div class="card__header">
+    <div class="card__price">
+      {{ticket.price}} Р
+    </div>
+    <img src="{{ticket.carrier | logo}}" alt="company" class="card__image">
+  </div>
+  <div class="card__race race race_first">
+    <div class="race__info info">
+      <div class="info__title">
+        {{ticket.segments[0].origin}} – {{ticket.segments[0].destination}}
+      </div>
+      <div class="info__data">
+        {{ticket.segments[0].date | date:'HH.mm'}}
```

Comment:
It's better for perfomance to create a variable for **ticket.segments[0]** and then call fields, which you need